src/components/component_base.cpp

```diff
@@ -11,14 +11,14 @@
 
 using viam::common::v1::ResourceName;
 
-ResourceName ComponentBase::get_resource_name(std::string name_) {
+ResourceName ComponentBase::get_resource_name(std::string name) {
```

Comment:
in this code base what are you having the trailing `_` denote?

Reply:
Good question! Would like to have a trailing `_` for all private data members of classes. Still a WIP getting to that point :)